skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -448,6 +449,160 @@ def _compute_metric_scores(
             return score[0] if score.size == 1 else score
         return score
 
+    @available_if(
+        lambda accessor: accessor._parent._timings_cache.get(("fit_time",)) is not None
+    )
+    def fit_time(self) -> float:
+        """Compute the time taken to fit the estimator.
```

Comment:
```suggestion
        """Renders the time taken to fit the estimator (in seconds).
```

I'd say renders because it's not actually computing anything.